nifty-core/src/main/java/com/facebook/nifty/ssl/SslSession.java

```diff
@@ -65,4 +65,13 @@ public long getEstablishedTime() {
     public X509Certificate getPeerCert() {
         return peerCert;
     }
+
+    public String toString() {
+        return "SslSession(alpn=" + (alpn == null ? "null" : alpn) +
```

Comment:
There is no need to do check for null, right? I mean you could just write something like  return "SslSession(alpn=" + alpn +...